src/bson.rs

```diff
@@ -111,9 +111,12 @@ impl Debug for Bson {
 
                 write!(f, "TimeStamp({}, {})", time, inc)
             }
-            Bson::Binary(t, ref vec) => {
-                write!(f, "BinData({}, 0x{})", u8::from(t), hex::encode(vec))
-            }
+            Bson::Binary(Binary { subtype, ref bytes }) => write!(
+                f,
+                "BinData({}, 0x{})",
```

Comment:
Probably makes sense to just write `Binary` here instead. I think `BinData` is just the shell's output for this type. Also, if this is updated to be like all the others (e.g. `CaseName({:?})`), then we can just do away with this `Debug` implementation altogether.

Reply:
I'll add a follow-up commit for this after I update the timestamp stuff as well (since that's currently being formatted as two integers like the new structure will be.